### PR TITLE
Service: Color

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/color.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Registers
+
+    ro color @ reading {
+        red: u8
+        green: u8
+        blue: u8
+    }
+
+Detected color in the RGB color space.
```
Comment (mmoskal) [inline services/color.md:16]: maybe do `u16 /` ?

Comment (pelikhan) [reply services/color.md:16]: 16 bits for each color channels?

Comment (mmoskal) [reply services/color.md:16]: yes; there are these color sensors for calibrating photo machines, I suspect they are more than 8 bit per channel

Comment (pelikhan) [reply services/color.md:16]: fixed.

**Review by mmoskal — APPROVED** [review]

**Pull request merged**